pkg/kv/bulk/sst_batcher.go

```diff
@@ -171,7 +181,11 @@ type SSTBatcher struct {
 
 		maxWriteTS hlc.Timestamp
 		totalRows  roachpb.BulkOpSummary
+		totalStats backuppb.IngestionPerformanceStats
```

Comment:
nit: Can you add a comment here about when totalStats is aggregated in the batcher